lib/rules/no-arguments-for-html-elements.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGES = {
+  1: 'Arguments ("%") should not be used on HTML elements ("<%>").',
+  2: 'Attribute value "%" won\'t work as expected, you need to wrap it as "%".',
+  3: 'Block params ("%") should not be used on HTML elements.',
+};
+
+function makeError(msgId, ...args) {
+  return args.reduce((result, value) => {
+    return result.replace('%', value);
+  }, ERROR_MESSAGES[msgId]);
+}
+module.exports = class NoAction extends Rule {
+  visitor() {
+    const isLocal = this.isLocal.bind(this);
+    const log = this.log.bind(this);
+    const sourceForNode = this.sourceForNode.bind(this);
+
+    function looksLikeHTMLElement(node) {
+      if (isLocal(node)) {
+        return false;
+      }
+      const tagName = node.tag;
+      if (tagName.startsWith('@') || tagName.startsWith(':') || tagName.includes('.')) {
+        return false;
+      }
+      return tagName.charAt(0) === tagName.charAt(0).toLowerCase();
+    }
+
+    function detectInvalidArgumentValues(attr) {
+      const name = attr.name;
+      const value = attr.value;
+      const originalNode = sourceForNode(value);
+      const safeValues = ['`', "'", '"'].reduce((result, wrapper) => {
+        result.push(wrapper + value.chars + wrapper);
+        return result;
+      }, []);
+      if (!safeValues.includes(originalNode)) {
+        log({
+          message: makeError(2, `${value.chars}`, `${name}={{${value.chars}}}`),
+          line: value.loc && value.loc.start.line,
+          column: value.loc && value.loc.start.column,
+          source: sourceForNode(value),
+        });
+      }
+    }
```

Comment:
I don't fully understand the point of this set of checks, can you explain?

Note, we already have [no-quoteless-attributes](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-quoteless-attributes.md) in the recommended rule set.

Reply:
good catch! removed